deploy/common/sensor-local-patch.yaml

```diff
@@ -0,0 +1,9 @@
+spec:
+  template:
+    spec:
+      containers:
+      - name: sensor
+        resources:
+          requests:
+            cpu: 500m
+            memory: 500Mi
```

Comment:
Should we change memory request to limit or because it's local patch we actually want to be able to schedule it and let it use more memory then requested

Reply:
The problem is that for `race-condition-qa-e2e-tests` - memory utilization goes quite high `<4GB`. If we want to specify mem limits, then we need two sets of configurations (local+ci and race-ci). I would avoid that if possible.

With the configuration changes proposed in this PR, we may encounter issues where a node does not have sufficient memory during pod runtime. If that happens, we need to define `request=limit` and have two sets. 🤷